userspace/libsinsp/filterchecks.cpp

```diff
@@ -3164,7 +3167,10 @@ uint8_t* sinsp_filter_check_event::extract(sinsp_evt *evt, OUT uint32_t* len, bo
 				evname = (uint8_t*)evt->get_name();
 			}
 
-			*len = strlen((char *) evname);
+			if(evname)
```

Comment:
Have you found cases where `evname` is null? Because as far as I can see evname will be always initialised from the if branches above.